### PR TITLE
Add HUAWEI CLOUD to cloud-provider vendor list

### PR DESCRIPTION
**Pull request by RainbowMango**
Add `HUAWEI CLOUD` to cloud-provider vendor list.

Fixes: https://github.com/kubernetes-sigs/cloud-provider-huaweicloud/issues/58

### CODE REVIEW

**Comment by RainbowMango** [comment]
/assign @onlydole
You can review the page at [here](https://deploy-preview-20453--kubernetes-io-master-staging.netlify.app/docs/concepts/architecture/cloud-controller/#vendor-implementations).

cc @kevin-wangzefeng

File: content/en/docs/concepts/architecture/cloud-controller.md [hunk 1]
```diff
@@ -231,6 +231,7 @@ The following cloud providers have implemented CCMs:
 * [DigitalOcean](https://github.com/digitalocean/digitalocean-cloud-controller-manager)
 * [GCP](https://github.com/kubernetes/cloud-provider-gcp)
 * [Hetzner](https://github.com/hetznercloud/hcloud-cloud-controller-manager)
+* [HUAWEI CLOUD](https://github.com/kubernetes-sigs/cloud-provider-huaweicloud)
```
Comment (jimangel) [inline content/en/docs/concepts/architecture/cloud-controller.md:234]: Should the link text be "HUAWEI Cloud" based on branding here: https://cloud.huawei.com/?

Comment (onlydole) [inline content/en/docs/concepts/architecture/cloud-controller.md:234]:
```suggestion
* [HUAWEI Cloud](https://github.com/kubernetes-sigs/cloud-provider-huaweicloud)
```

**Comment by RainbowMango** [comment]
@jimangel @onlydole 
Thanks. But the brand should base on www.huaweicloud.com.
`cloud.huawei.com` is mainly for Huawei Mobile Services.

**Comment by jimangel** [comment]
> the brand should base on www.huaweicloud.com.

It does appear to be all in CAPs :smile: 

/lgtm
/approve

**Pull request merged**